system-tests/lib/normalizeStdout.ts

```diff
@@ -166,6 +166,9 @@ export const normalizeStdout = function (str: string, options: any = {}) {
   .replace(/Still waiting to connect to .+, retrying in 1 second \(attempt .+\/.+\)\n/g, '')
   // Replaces CDP connection error message in Firefox since Cypress will retry
   .replace(/\nFailed to spawn CDP with Firefox. Retrying.*\.\.\.\n/g, '')
+  // Replaces "new dependencies optimized" message from vite as it does not respect the logLevel='silent' option
+  .replace(/^.*Re-optimizing dependencies.*?\n$/gm, '')
+  .replace(/\).*new dependencies optimized.*?\n/gm, ')\n')
```

Comment:
Is this going to annoy users?

Reply:
In this case no. It only happens when the dependency tree of an import is updated, which happens in our system tests when executed the firsttime because the install is fresh. In other words, this isn't a common occurance